.travis.yml

```diff
@@ -432,3 +432,4 @@ jobs:
 branches:
   only:
     - master
+    - wilhuff/swift-integration
```

Comment:
Intended?

Reply:
Yes -- that was the last commit and it enables testing in travis. Part of resolving and removing the diffbase for the PR will be to remove it. Without this no tests will run until after the PR is retargeted at master.